app/build.gradle

```diff
@@ -20,8 +20,7 @@ android {
             cmake {
                 cppFlags "-std=c++11"
```

Comment:
Don't commit this changes to build.gradle please. Make it your own stash for now